projects/vdk-plugins/vdk-impala/tests/functional/jobs/insert_template_partition_job/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2021 VMware, Inc.
```

Comment:
Please remove the empty `__init__.py` files. I think those are created accidentally by VSCode